src/iter/parallel.rs

```diff
@@ -0,0 +1,135 @@
+use rayon::iter::ParallelIterator;
+use rayon::iter::internal::{UnindexedProducer, UnindexedConsumer, Folder, bridge_unindexed};
+use super::*;
```

Comment:
let's not do this, considered harmful in Rust style to be used anywhere outside of test modules